engine/src/main/java/org/hibernate/search/cfg/spi/SearchConfigurationBase.java

```diff
@@ -55,4 +55,9 @@ public InstanceInitializer getInstanceInitializer() {
 	public boolean isIdProvidedImplicit() {
 		return false;
 	}
+
+	@Override
+	public boolean isMultitenancyEnabled() {
+		return true;
```

Comment:
Why true in this case? It should probably be false, but then I'm slightly concerned that this PR actually breacks things when the tenant_id is not part of the schema, as because of this line we're often testing only the "true" case